### PR TITLE
Add new dev environment

### PR DESCRIPTION
**Pull request by dantheman39**

### CODE REVIEW

**Review by shughes-uk — APPROVED** [review]

**Review by marcosmoyano — APPROVED** [review]

**Comment by dantheman39** [comment]
Blocked by this: https://gitlab.com/coiled/cloud/-/issues/3604 . I can see about creating the repo manually to unblock this

**Pull request merged**
